app/dev_mode/templates/dev-page.html

```diff
@@ -156,6 +156,14 @@
              </div>
 
             <div class="field-container">
+              <label for="region_code">Language</label>
```

Comment:
Minor, and only affecting the Dev page, but label is for the wrong element, should be `for="language_code"`.